account_analytic_parent/wizard/account_analytic_wizard.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2017 Tecnativa - Vicent Cubells
+# Copyright 2017 Tecnativa - David Vidal
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import api, fields, models
+
+
+class AccountAnalyticChart(models.TransientModel):
+    _name = "account.analytic.chart"
+    _description = "Account Analytic Chart"
+
+    from_date = fields.Date(string="From")
+    to_date = fields.Date(string="To")
+
+    @api.multi
+    def analytic_account_chart_open_window(self):
```

Comment:
Is this still maintained?
When testing, this wizard opened a form view to create a new Analytic Account. I don't think it is the expected behavior.

I was expecting to get a list of analytic accounts with the debit,credit,balance for the period selected in the wizard.

Reply:
Hierarchical tree view is no more: [GitHub thread](https://github.com/odoo/odoo/issues/20659).

I can not think of a way to replicate the hierarchical view. So, maybe just get rid of the wizard, the report view and action and the test?

Reply:
Since hierarchical tree view has gone, I think also it would be better to discard this functionality and not to be maintained anymore.
If the community agree, I'll discard this useless part of the code in the next days, so this PR could be approved and merged.